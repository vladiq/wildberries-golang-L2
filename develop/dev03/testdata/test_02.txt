qqqqqqq tttttttt ssssss
skd aaaaa rrrrrrr
234 4324 234432
fffff aaaaaa ttttttt