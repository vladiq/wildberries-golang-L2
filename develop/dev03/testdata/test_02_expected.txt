234 4324 234432
skd aaaaa rrrrrrr
fffff aaaaaa ttttttt
qqqqqqq tttttttt ssssss
